app/controllers/serious_misconducts_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SeriousMisconductsController < ApplicationController
```

Comment:
Should this be singular?

```suggestion
class SeriousMisconductController < ApplicationController
```